base/linalg/arnoldi.jl

```diff
@@ -354,7 +354,10 @@ iterations derived from [`eigs`](@ref).
 
 **Outputs**
 
-* `svd`: An `SVD` object containing the left singular vectors, the requested values, and the right singular vectors. If `ritzvec = false`, the left and right singular vectors will be empty.
+* `svd`: An `SVD` object containing the left singular vectors, the requested values, and the right singular vectors.
+If `ritzvec = false`, the left and right singular vectors will be empty. U, S, V and Vt can be obtained from the SVD
+object with `Z[:U]`, `Z[:S]`, `Z[:V]` and `Z[:Vt]` (where `Z = svds(A)[1]`), such that `A = U*diagm(S)*Vt`.
+Internally Vt is stored and hence Vt is more efficient to extract than V, because in contrast to LAPACK, which produces Vt, the algorithm used by ARPACK actually produces V.
```

Comment:
That makes the sentence even more confusing since it says:

1. `Vt` is more efficient to extract than `V`
2. the algorithm used by ARPACK actually produces `V`

This seems to be claiming that we store `Vt` but the algorithm we use produces `V`. Which is it? Also, this could stand to clarify which library we use for which computations – dense versus sparse – this isn't general knowledge for every reader.